app/controllers/concerns/pubmed_search.rb

```diff
@@ -2,8 +2,8 @@ module PubmedSearch
   extend ActiveSupport::Concern
 
   def search_pubmed(search_term)
-    @search_terms = search_term.split.join("+")
-    search_url = 'http://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?db=pubmed&retmode=json&retmax=20&term='+@search_terms
+    @search_term = search_term.split.join("+")
+    search_url = 'http://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?db=pubmed&retmode=json&retmax=20&term='+@search_term
```

Comment:
There's probably something in Net/HTTP that handles appending parameters, come to think of it. That would be dumb not to have.